app/controllers/PaymentController.scala

```diff
@@ -25,38 +26,58 @@ class PaymentController @Inject()(ws:WSClient,cc: ControllerComponents) extends
       accessToken
     }
 
-  def createOrder() ={
-
+  def createOrder() ={//returns the order ID
     val json: JsValue = Json.parse("""
   {
+    "application_context":{
+        "return_url":"http://localhost:9099/capturePayment"
+    },
     "intent": "CAPTURE",
     "purchase_units": [
     {
       "reference_id": "TICKET",
       "amount": {
         "currency_code": "GBP",
-        "value": "240.00"
+        "value": "69.00"
       }
     }
   ]
   }
   """)
     val token="Bearer "+getAccessToken()
-    val request=ws.url("https://api.sandbox.paypal.com/v2/checkout/orders").
+    Await.result(ws.url("https://api.sandbox.paypal.com/v2/checkout/orders").
       addHttpHeaders("Content-Type"->"application/json").
       addHttpHeaders("Authorization"->token).
-      post(json)
-      request//this should be removed
+      post(json),Duration(5,"seconds")).json
+
   }
-  def lol: Unit ={
-    val request=ws.url("https://api.sandbox.paypal.com/v2/payments/authorizations/")
+  def capturePayment(orderID:String)=Action{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method takes a value that is controlled by the client](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5261906)